.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Issue report
+about: Report an issue to help us improve the devguide
+---
+
+<!--
+Note: This repo is for the Python devguide. If you are reporting an issue or
+enhancement for the standard library, then the CPython issue tracker is better
```

Comment:
Why is the stdlib specifically called out compared to e.g. the language or CPython interpreter?

Reply:
No particular reason. Will replace with "language".